docs/en/04_Changelogs/4.4.0.md

```diff
@@ -73,9 +73,9 @@ Here's an example of how file names changed during an initial 4.x migration, and
  * File migrated under SS 4.x with `legacy_filenames=false`: `assets/[content-hash]/myfile.pdf` (Regression: links to `assets/myfile.pdf` no longer work)
  * File migrated under SS 4.x with `legacy_filenames=true`: `assets/myfile.pdf`
  * File with updated file content under SS 4.x: `assets/[new-content-hash]/myfile.pdf` (Regression: links to `assets/[content-hash]/myfile.pdf` no longer work)
- * File with hotfix applied (SS 4.2.3): `assets/[new-content-hash]/myfile.pdf` (redirects to `assets/myfile.pdf`)
- * File with full fix applied (SS 4.3.x) : `assets/myfile.pdf` (no redirect required)
- * Newly uploaded file with full fix applied: `assets/my-other-file.pdf` (no redirect required)
+ * File with hotfix applied (SS 4.2.3): `assets/myfile.pdf` and `assets/[old-content-hash]/myfile.pdf` redirects to `assets/[new-content-hash]/myfile.pdf`
```

Comment:
The point here was to describe the location of the file on the filesystem, not the different virtual or physical paths which get redirected. Those are two different pieces of info, you can't replace one with the other